packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -438,6 +439,7 @@ export class DetailsListBase extends React.Component<IDetailsListProps, IDetails
                   ariaLabel: ariaLabelForListHeader,
                   ariaLabelForSelectAllCheckbox: ariaLabelForSelectAllCheckbox,
                   ariaLabelForSelectionColumn: ariaLabelForSelectionColumn,
+                  ariaLabelForToggleAllGroupsButton: detailsHeaderProps && detailsHeaderProps.ariaLabelForToggleAllGroupsButton,
```

Comment:
Perhaps this needs to be revised in the future with a spread at a very well thought position to allow and limit what things a user could override.